pages/common/dvc-unfreeze.md

```diff
@@ -0,0 +1,13 @@
+# dvc unfreeze
+
+> Unfreeze stages in DVC pipeline.
+> This makes DVC to start tracking changes in stage dependencies again after they were frozen.
+> More information: <https://dvc.org/doc/command-reference/unfreeze>.
```

Comment:
Perhaps this will read better?
```suggestion
> Unfreeze stages in the DVC pipeline.
> This allows DVC to start tracking changes in stage dependencies again after they were frozen.
> See also `dvc freeze`.
> More information: <https://dvc.org/doc/command-reference/unfreeze>.
```